### PR TITLE
Add meta descriptions

### PR DESCRIPTION
**Pull request by matt-paul**
#### Trello Ticket / Github Issue Link
https://trello.com/c/ilnCfUqZ
https://docs.google.com/document/d/1MOeXI9RVxF86sWCQHoNs3deT58fZMsmQdghE4ZZ6Tks/edit
#### Description
Meta descriptions have not been added to the site pages.  This fixes them for the following pages....
 - Home
 - What we do
 - Our work
 - About Us
 - Meet our Team
 - Jobs

#### Can this PR be merged immediately after testing (eg. rebase, infrastructure update)?
Yes

### CODE REVIEW

File: site/index.ejs [hunk 1]
```diff
@@ -1,90 +1,113 @@
...
+        <% if (typeof cssPath !== 'undefined') { %>
+          <link rel="stylesheet" href="<%= cssPath %>" type="text/css" charset="utf-8">
+          <% } %>
+            <% if (typeof tracking !== 'undefined' && tracking) { %>
+              <script async type="text/javascript" src="https://secure.leadforensics.com/js/77932.js"></script>
+              <noscript>
+                <img src="https://secure.leadforensics.com/77932.png" style="display: none;" />
+              </noscript>
+              <% } %>
+                <script>
```
Comment (dpiatek) [inline site/index.ejs]: This formatting is actually wrong (for title and the script after it) ... don't think prettier handles ejs files well. Maybe we shouldn't be running prettier on this file, can we ignore it in config?

File: site/index.ejs [hunk 2]
```diff
@@ -1,90 +1,113 @@
...
+                  <img height="1" width="1" style="display:none;" alt="" src="https://dc.ads.linkedin.com/collect/?pid=103577&fmt=gif" />
+                </noscript>
+                <!-- End LinkedIn Pixel Code -->
+</head>
+
+<body>
+  <div class="js-app">
+    <% if (typeof bodyContent !== 'undefined') { %>
+      <%= bodyContent %>
+        <% } %>
```
Comment (dpiatek) [inline site/index.ejs]: Same comment as above

**Comment by dpiatek** [comment]
Needs a rebase 🙏

File: site/index.ejs (hunk 1 above)
Comment (matt-paul) [reply site/index.ejs]: added .prettierignore and excluded ejs files

File: site/routes/index.js [hunk 3]
```diff
@@ -132,8 +132,16 @@ const handleContactUsHash = stateNavigator => {
 export default () => {
   const stateNavigator = new StateNavigator(
     routes(),
-    new HTML5HistoryManager((process.env.URL_BASENAME || '').slice(0, -1)),
+    new HTML5HistoryManager((process.env.URL_BASENAME || '').slice(0, -1))
   );
   handleContactUsHash(stateNavigator);
+  stateNavigator.onNavigate((oldState, state) => {
+    const page = routeDefinitions.find(obj => obj.key === state.key);
+    if (typeof window !== 'undefined') {
```
Comment (dpiatek) [inline site/routes/index.js]: should't this be `document` instead of `window` ?

**Pull request merged**